src/components/IssueUI/WhoopsStatusUI/index.tsx

```diff
@@ -68,10 +74,10 @@ const WhoopsStatusUI = ({ request }: Props): JSX.Element => {
         className='w-full'
         title={<h5 className={getColorShade('yellow')}>{t('issue_page.refund_deposited')}</h5>}
         unitIcon={<BitcoinLogoIcon width={23} height={23} />}
-        value={displayMonetaryAmount(request.backingPayment.amount)}
+        value={displayMonetaryAmount(monetaryBackingPaymentAmount)}
```

Comment:
There was a bug here.
`request.backingPayment.amount` is a numerical atomic value but passed as a monetary value which breaks the UI. So I fixed it.
<img width="1496" alt="Screenshot 2022-12-05 at 2 04 08 AM" src="https://user-images.githubusercontent.com/84005068/205527896-2da234ff-842b-416f-b779-78258797f72a.png">